source/accessibility-statement.html.erb

```diff
@@ -48,31 +69,59 @@ title: Accessibility statement for GOV.UK Pay
       <p class="govuk-body">The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you are not happy with how we respond to your complaint, <a class="govuk-link" href="https://www.equalityadvisoryservice.com/">contact the Equality Advisory and Support Service</a>, which is run on behalf of EHRC.</p>
 
       <h2 class="govuk-heading-m">Technical information about this service’s accessibility</h2>
-      <p class="govuk-body">We’re committed to making this service accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018. </p>
-      <p class="govuk-body">This service is fully compliant with the Web Content Accessibility Guidelines version 2.1 AA standard.</p>
+      <p class="govuk-body">We’re committed to making this service accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.</p>
+      <p class="govuk-body">This service is partially compliant with the <a class="govuk-link" href="https://www.w3.org/TR/WCAG21/">Web Content Accessibility Guidelines version 2.1 AA standard</a>, due to the non-compliances listed below.</p>
 
-      <h2 class="govuk-heading-m">How we tested this service</h2>
-      <p class="govuk-body">We last tested this service for compliance with WCAG 2.1 AA on 5 September 2019. </p>
-      <p class="govuk-body">We used this approach to deciding on a sample of pages to test.</p>
+      <h2 class="govuk-heading-m">Non accessible content</h2>
+      <p class="govuk-body">The content listed below is non-accessible for the following reasons.</p>
+
+      <h3 class="govuk-heading-s">Non compliance with the accessibility regulations</h3>
+      <p class="govuk-body">For paying users:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>error messages and labels for card expiry dates may not be announced to screen readers (WCAG success criterion 4.1.3)</li>
```

Comment:
Can we add something like `card numbers` and `card expiry dates` here?
As it would also cover client side error messages to do with the card number.

Reply:
Thanks, I've taken that in.